VimeoNetworking/VimeoNetworking/Request+Authentication.swift

```diff
@@ -219,9 +233,9 @@ extension Request where ModelType: PinCodeInfo
      */
     static func getPinCodeRequest(scopes scopes: [Scope]) -> Request
     {
-        let parameters: VimeoClient.RequestParameters = [GrantTypeKey: GrantTypePinCode,
```

Comment:
What is the advantage / disadvantage of having this `VimeoClient.RequestParameters` type specified here?